frontend-v2/README.md

```diff
@@ -5,13 +5,28 @@ This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-
 See the main project README for initial, one-time setup as well as instructions
 to build the project and run the development servers.
 
-## Learn More
+## Frontend migration
+
+We have two frontend applications that are built independently except that the
+React app imports a single component from the Vue app--the navigation bar.
+This allows us to define in one place which pages belong to which application.
+
+Each application is hosted at the same origin, so cookies are shared.
+
+All React pages are served under the path `v2/*`  (via a proxy in local dev so
+the next.js dev server can properly run, then via static files in the prod
```

Comment:
It's actually a proxy in prod now too so that Next.js can do SSR for smoother page loads.

Reply:
Updated